.gitignore

```diff
@@ -5,6 +5,7 @@
 .cache/
 .pytest_cache/
 src/test_files
```

Comment:
Basically, you can just not add these files to the git repository, so these two lines can be dropped. For instance, don't run `git add src/test_files` and `git add src/data.json` or `git add .`

Reply:
deleted, done